README.rst

```diff
@@ -52,3 +52,23 @@ Comparing ``rdopkg patch`` with this ``tar-changes`` script:
 
 .. _rdopkg: https://github.com/softwarefactory-project/rdopkg
 .. _Limited: https://github.com/softwarefactory-project/rdopkg/issues/169
+
+##Understanding branches
```

Comment:
This README is reStructuredText, not Markdown, so the syntax is slightly different:

```suggestion
Understanding branches
----------------------
```